remote-monitoring/docker-compose.dotnet.yml

```diff
@@ -32,14 +32,20 @@ services:
     environment:
       - PCS_AUTH_ISSUER
       - PCS_AUTH_AUDIENCE
+      - PCS_APPLICATION_SECRET
 
   iothubmanager:
     image: azureiotpcs/iothub-manager-dotnet
     depends_on:
-      - uiconfig
+      - config
     environment:
       - PCS_IOTHUB_CONNSTRING
-      - PCS_UICONFIG_WEBSERVICE_URL=http://uiconfig:9005/v1
+      # TODO: remove this and keep only PCS_CONFIG_WEBSERVICE_URL
+      - PCS_UICONFIG_WEBSERVICE_URL=http://config:9005/v1
+      - PCS_CONFIG_WEBSERVICE_URL=http://config:9005/v1
+      - PCS_AUTH_ISSUER
+      - PCS_AUTH_AUDIENCE
+      - PCS_APPLICATION_SECRET
```

Comment:
Will need to add all this new env vars to k8s .yaml as well in pcs-cli repo

Reply:
I was planning to do it, unless you prefer doing it on your side